src/pages/StockListing/StockListing.tsx

```diff
@@ -24,12 +24,32 @@ const StockListing = (): JSX.Element => {
   const howManyToDisplay: number = 10;
 
   useEffect(() => {
-    const fetchData = async () => {
+    // check for logged on User
+    const loggedUser = window.localStorage.getItem('loggedOnUser')
+    if(loggedUser){
+      const fetchUserData = async () =>{
+        const rawData = await fetch(`${process.env.REACT_APP_BACKEND_URL}/api/v1/stocks`,
+        {
+          method: "GET",
+          credentials: 'include',
+        })
+        
+        if(rawData.ok){
+          const data = await rawData.json()
+          setData(data.stocks)
+          fetchUserData();
```

Comment:
When I'm logged in I see the stocks, but I think this line is causing an infinite loop where it keeps making requests to `/api/v1/stocks`. I tried deleting it, but I see that this then makes it so I can't see the stocks. I'm not immediately sure how to fix this, but I'm thinking it is likely to be solved if we pass down the logged in state?